tools/pipelines/templates/build-npm-package.yml

```diff
@@ -4,601 +4,645 @@
+              # this is used to inject telemetry key file and upload built devtools extension.
+              - ${{ parameters.additionalBuildSteps }}
+
+            templateContext:
+              outputParentDirectory: $(Build.ArtifactStagingDirectory)
+              outputs:
+                - ${{ if and(ne(convertToJson(parameters.taskTest), '[]'), contains(convertToJson(parameters.taskTest), 'tinylicious')) }}:
+                    - output: pipelineArtifact
+                      displayName: Publish Artifact - Tinylicious Log
+                      condition: and(succeededOrFailed(), eq(variables['startTest'], 'true'))
+                      targetPath: '$(Build.ArtifactStagingDirectory)/tinyliciousLog/tinylicious.log'
+                      artifactName: tinyliciousLog_attempt-$(System.JobAttempt)
+                      sbomEnabled: false
+                      publishLocation: pipeline
+                      continueOnError: true # Keep running subsequent tasks even if this one fails (e.g. the tinylicious log wasn't there)
+
+                - ${{ if ne(parameters.taskPack, false) }}:
+                    - output: pipelineArtifact
+                      displayName: Publish Artifact - pack
+                      targetPath: $(Build.ArtifactStagingDirectory)/pack
+                      artifactName: pack
+                      publishLocation: pipeline
+
+                    - output: pipelineArtifact
+                      displayName: Publish Artifact - Test Files
+                      targetPath: $(Build.ArtifactStagingDirectory)/test-files
+                      artifactName: test-files
+                      publishLocation: pipeline
+                      sbomEnabled: false
+
+                - ${{ if eq(parameters.taskBundleAnalysis, true) }}:
+                    - output: pipelineArtifact
+                      displayName: Publish Artifacts - bundle-analysis
+                      condition: and( succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(${{ parameters.taskPublishBundleSizeArtifacts }}, true) )
+                      targetPath: $(Build.ArtifactStagingDirectory)/bundleAnalysis
+                      artifactname: bundleAnalysis
+                      sbomEnabled: false
+                      publishLocation: pipeline
+
+                - ${{ if or(eq(parameters.publishDocs, true), eq(parameters.taskBuildDocs, true)) }}:
+                    - output: pipelineArtifact
+                      displayName: Publish Artifact - _api-extractor-temp
+                      targetPath: $(Build.ArtifactStagingDirectory)/_api-extractor-temp
+                      artifactName: _api-extractor-temp
+                      sbomEnabled: false
+                      publishLocation: pipeline
+
+          # Job - Component detection
+          - ${{ if eq(parameters.publish, true) }}:
+              - job: CG
+                displayName: Component Detection
+                pool: Small-1ES
```

Comment:
Let's restore the use of `{{ parameters.pollCG }}` here. With this change the parameter would become unnecessary (I'd argue it already is unnecessary because we don't override its default anywhere) but seems like we didn't clean that up in the "master" migration PR, so better handle this separately and for now we can keep using it here. Just keep the indentation fix.

Reply:
Updated, LMK if that looks right.

Reply:
did you mean to say this "poolCG" or pollCG?

Reply:
No, if you look at the diff the line to update was 588. We moved from a parameter for the pool name, to hardcoding Small-1ES. Line 585 needs to stick with `parameters.publish`. And my bad, typo in the previous comment, I think it's `poolCG`, not `pollCG` (you can confirm by looking at the parameters for the file).

Reply:
Updated.